Cargo.toml

```diff
@@ -57,6 +57,7 @@ too_many_arguments = "allow"
 nonstandard_macro_braces = "warn"
 print_stdout = "warn"
 print_stderr = "warn"
+unwrap_used = "warn"
```

Comment:
You may need to copy this to [a few other `Cargo.toml`s](https://github.com/search?q=repo%3Abevyengine%2Fbevy%20%22lints.clippy%22&type=code), since not all inherit from the workspace. (Notably, the main `bevy` package doesn't seem to inherit from the workspace.)